gensim/test/test_summarization.py

```diff
@@ -147,6 +147,22 @@ def test_keywords_runs(self):
 
         kwds_lst = keywords(text, split=True)
         self.assertTrue(len(kwds_lst))
+        
+    def test_mz_keywords(self):
```

Comment:
Need to add the more concrete tests (not only "sanity check"), I mean with the concrete words.